app-kotlin/src/main/kotlin/au/com/gridstone/trainingkotlin/Constants.kt

```diff
@@ -0,0 +1,11 @@
+package au.com.gridstone.trainingkotlin
+
+object Constants {
+  const val MY_TAG = "MY_LOG_TAG"
+
+  //  const val  BASE_URL = "https://pokeapi.co/api/v2/pokemon/"
+  const val BASE_URL = "https://httpstat.us/400/"
+  const val IMAGE_URL =
+    "https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/other/official-artwork"
+}
```

Comment:
These should belong in build configs

Reply:
thanks i will be doing this next after implementing detail screen by doing some research over internet

Reply:
Thanks i had fixed this.